### ISSUE TITLE
Incorrect window align 

### ISSUE THREAD

**Issue by capitanBOBO** [koekeishiya/yabai#1396]
After reload windows aligning incorrectly
![image](https://user-images.githubusercontent.com/29453809/188815345-be22a8ac-a37b-4728-a643-c2ceac83b558.png)

**Comment by capitanBOBO** [comment]
Monterey, Intel

**Comment by D-PXR** [comment]
Could you post your  err.log and yabairc?

**Comment by capitanBOBO** [comment]
> Could you post your err.log and yabairc?

SIP enabled. Sorry, I'm forgot where I can find err.log?
```
#!/usr/bin/env sh

# the scripting-addition must be loaded manually if
# you are running yabai on macOS Big Sur. Uncomment
# the following line to have the injection performed
# when the config is executed during startup.
#
# for this to work you must configure sudo such that
# it will be able to run the command without password
#
# see this wiki page for information:
#  - https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)
#
# sudo yabai --load-sa
# yabai -m signal --add event=dock_did_restart action="sudo yabai --load-sa"

# global settings
yabai -m config mouse_follows_focus          on
yabai -m config focus_follows_mouse          off
yabai -m config window_placement             second_child
yabai -m config window_topmost               on
yabai -m config window_shadow                on
yabai -m config window_opacity               off
yabai -m config window_opacity_duration      0.0
yabai -m config active_window_opacity        1.0
yabai -m config normal_window_opacity        0.90
yabai -m config window_border                off
yabai -m config window_border_width          6
yabai -m config active_window_border_color   0xffff79c6
yabai -m config normal_window_border_color   0xff555555
yabai -m config insert_feedback_color        0xffd75f5f
yabai -m config split_ratio                  0.50
yabai -m config auto_balance                 off
yabai -m config mouse_modifier               fn
yabai -m config mouse_action1                move
yabai -m config mouse_action2                resize
yabai -m config mouse_drop_action            swap

# general space settings
yabai -m config layout                       bsp
yabai -m config top_padding                  5
yabai -m config bottom_padding               5
yabai -m config left_padding                 5
yabai -m config right_padding                5
yabai -m config window_gap                   5
yabai -m config external_bar all:24:0

yabai -m rule --add app="System Preferences" sticky=on layer=above manage=off
yabai -m rule --add app="Системные настройки" sticky=on layer=above manage=off
yabai -m rule --add app="Google Chrome" manage=off
yabai -m rule --add app="^Safari$" manage=off
yabai -m rule --add app="^EndpointConnect$" manage=off
yabai -m rule --add app="^Cisco AnyConnect Secure Mobility Client$" manage=off
yabai -m rule --add app="^Rectangle$" sticky=on layer=above manage=off
yabai -m rule --add app="Digital Color Meter" sticky=on manage=off
yabai -m rule --add app="Цифровой колориметр" sticky=on manage=off
yabai -m rule --add app="Calendar" manage=off
yabai -m rule --add app="Календарь" manage=off
yabai -m rule --add app="Калькулятор" manage=off
yabai -m rule --add app="Calculator" manage=off
yabai -m rule --add app="Alacritty" manage=off
yabai -m rule --add app="Karabiner-Element" manage=off
yabai -m rule --add app="Simulator" manage=off
yabai -m rule --add app="Alfred Preference" manage=off
yabai -m rule --add app="App Store" manage=off
yabai -m rule --add app="Информация о системе" manage=off
yabai -m rule --add app="Riot Client" manage=off
yabai -m rule --add app="League of Legends" manage=off
yabai -m rule --add app="Генератор отчетов об ошибках" manage=off
yabai -m rule --add app="OpenEmu" manage=off
yabai -m rule --add app="WhatsApp" manage=off
yabai -m rule --add app="Команды" manage=off
yabai -m rule --add app="Sourcetree" manage=off

yabai -m signal --add event=application_front_switched \
	action="osascript -e 'tell application \"Übersicht\" to refresh widget id \"mybar-index-jsx\"'"
yabai -m signal --add event=space_changed \
	action="osascript -e 'tell application \"Übersicht\" to refresh widget id \"mybar-index-jsx\"'"
yabai -m signal --add event=display_changed \
	action="osascript -e 'tell application \"Übersicht\" to refresh widget id \"mybar-index-jsx\"'"
yabai -m signal --add event=window_focused \
	action="osascript -e 'tell application \"Übersicht\" to refresh widget id \"mybar-index-jsx\"'"
yabai -m signal --add event=window_destroyed \
	action="osascript -e 'tell application \"Übersicht\" to refresh widget id \"mybar-index-jsx\"'"

yabai -m space 1 --label www
yabai -m space 2 --label dev
yabai -m space 3 --label chat
yabai -m space 4 --label other

echo "yabai configuration loaded.."

```

**Comment by capitanBOBO** [comment]
> Could you post your err.log and yabairc?
`'window_opacity_duration' cannot be changed on macOS Catalina/Big Sur/Monterey because of an Apple bug in the WindowServer`

Found it

**Comment by capitanBOBO** [comment]
Removed `window_opacity_duration`, don't have any errors but problem still here

**Comment by D-PXR** [comment]
> Removed `window_opacity_duration`, don't have any errors but problem still here

Is only a specific app doing this, or are all apps acting like this?

**Comment by capitanBOBO** [comment]
> Is only a specific app doing this, or are all apps acting like this?

Some of them, not specific. It's can be xcode, brave-browser, telegram
But, I think it's always the left-most window

**Comment by capitanBOBO** [comment]
And another moment. After I fixing window position by mouse, it's sets everything correctly

**Comment by D-PXR** [comment]
> And another moment. After I fixing window position by mouse, it's sets everything correctly

Try turning 

`yabai -m config mouse_follows_focus          off`
and 
`yabai -m config focus_follows_mouse          off`

See if that does anything.

**Comment by capitanBOBO** [comment]
> > And another moment. After I fixing window position by mouse, it's sets everything correctly
> 
> Try turning
> 
> `yabai -m config mouse_follows_focus off` and `yabai -m config focus_follows_mouse off`
> 
> See if that does anything.

no, nothing changed

**Comment by D-PXR** [comment]
> > > And another moment. After I fixing window position by mouse, it's sets everything correctly
> > 
> > 
> > Try turning
> > `yabai -m config mouse_follows_focus off` and `yabai -m config focus_follows_mouse off`
> > See if that does anything.
> 
> no, nothing changed

Could you try changing
`yabai -m config window_placement             second_child` to `yabai -m config window_placement             first_child`

**Comment by capitanBOBO** [comment]
> > > > And another moment. After I fixing window position by mouse, it's sets everything correctly
> > > 
> > > 
> > > Try turning
> > > `yabai -m config mouse_follows_focus off` and `yabai -m config focus_follows_mouse off`
> > > See if that does anything.
> > 
> > 
> > no, nothing changed
> 
> Could you try changing `yabai -m config window_placement second_child` to `yabai -m config window_placement first_child`

nope, didn't help

**Comment by koekeishiya** [comment]
This is an issue related to some caching mechanism I am experimenting with on the master branch: https://github.com/koekeishiya/yabai/commit/c20adae78d721f65db6cb3b6c9dc55310f9c1633

Use the released v4.0.2 instead.

**Comment by capitanBOBO** [comment]
> This is an issue related to some caching mechanism I am experimenting with on the master branch: [c20adae](https://github.com/koekeishiya/yabai/commit/c20adae78d721f65db6cb3b6c9dc55310f9c1633)
> 
> Use the released v4.0.2 instead if this bothers you.

Thanks! I'll try

**Comment by koekeishiya** [comment]
reverted in commit https://github.com/koekeishiya/yabai/commit/a64df0c8978c2ee27f4a9fdfbba13ea3bc6ead19
